pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+
+        References
+        ----------
+
+        >>> oEditor.GetMatchedObjectName
+        """
+        return self.oeditor.GetMatchedObjectName(search_string)
+
+    @pyaedt_function_handler()
+    def clean_objects_name(self, main_part_name):
+        """Clean the names of the objects for a main part.
+
+        Parameters
+        ----------
+        main_part_name : str
+            Name of the main part.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+
+        References
+        ----------
+
+        >>> oEditor.RenamePart
+        """
+        # import os.path
+        # (CADPath, CADFilename) = os.path.split(CADFile)
+        # (CADName, CADExt) = os.path.splitext(CADFilename)
+        CADSuffix = main_part_name + "_"
+        objNames = self.oeditor.GetMatchedObjectName(CADSuffix + "*")
+        for name in objNames:
+            RenameArgs = {}
+            RenameArgs["NAME"] = "Rename Data"
+            RenameArgs["Old Name"] = name
+            RenameArgs["New Name"] = name.replace(CADSuffix, "")
+            self.oeditor.RenamePart(RenameArgs)
+        return True
+
+    @pyaedt_function_handler()
+    def create_airbox(self, offset=0, offset_type="Absolute", defname="AirBox_Auto"):
+        """Create an airbox that is as big as the bounding extension of the project.
+
+        Parameters
+        ----------
+        offset :
+            Double offset value to apply on the airbox faces versus the bounding box.
+            The default is ``0``.
+        offset_type : str
+            Type of the offset. Options are ``"Absolute"`` and ``"Relative"``.
+            The default is ``"Absolute"``. If ``"Relative"``, the offset input
+            is between 0 and 100.
+        defname : str, optional
+            Name of the airbox. The default is ``"AirBox_Auto"``.
+
+        Returns
+        -------
+        int
+            ID of the airbox created.
+
+        References
+        ----------
+
+        >>> oEditor.CreateBox
+        """
+        self.logger.info("Adding Airbox to the Bounding ")
+
+        bound = self.get_model_bounding_box()
+        if offset_type == "Absolute":
+            offset1 = offset2 = offset3 = offset
+        else:
+            offset1 = (bound[3] - bound[0]) * offset / 100
+            offset2 = (bound[4] - bound[1]) * offset / 100
+            offset3 = (bound[5] - bound[2]) * offset / 100
+        startpos = self.Position(bound[0] - offset1, bound[1] - offset2, bound[2] - offset3)
+
+        dim = []
+        dim.append(bound[3] - bound[0] + 2 * offset1)
+        dim.append(bound[4] - bound[1] + 2 * offset2)
+        dim.append(bound[5] - bound[2] + 2 * offset3)
+        airid = self.create_box(startpos, dim, defname)
+        return airid
+
+    @pyaedt_function_handler()
+    def create_air_region(self, x_pos=0, y_pos=0, z_pos=0, x_neg=0, y_neg=0, z_neg=0, is_percentage=True):
+        """Create an air region.
+
+        Parameters
```

Comment:
All of these parameters should have a brief description before the "If" statements. Perhaps the "... in modeler units: should be part of this brief description so as not to seem exclusive to a float value. Spell out string in these statements.